src/modules/addReviewConfidence.js

```diff
@@ -1,28 +1,80 @@
-//not useful?
-function addReviewConfidence(){
-	generalAPIcall("query{Page(page:1,perPage:30){reviews(sort:ID_DESC){id rating ratingAmount}}}",{},function(data){
-		let adder = function(){
-			if(location.pathname !== "/reviews"){
-				return
+exportModule({
+	id: "reviewConfidence",
+	description: "Add confidence scores to reviews",
+	isDefault: true,
+	categories: ["Browse"],
+	visible: true,
+	urlMatch: function(url){
+		return /^https:\/\/anilist\.co\/reviews/.test(url)
+	},
+	code: function(){
+		let pageCount = 0;
+		const adultContent = userObject ? userObject.options.displayAdultContent : false;
+
+		function watchElem(selector, parent) {
+			return new Promise(resolve => {
+				new MutationObserver((_mutations, observer) => {
+					const elem = (parent || document).querySelector(selector);
+					if (elem) {
+						observer.disconnect()
+						resolve(elem)
+					}
+				}).observe(document.body, { subtree: true, childList: true })
```

Comment:
Isn't watching the subtree and childlist of ``document.body`` very expensive? That is, this will fire on every page modification. Can perhaps document.body be reduced to some more narrow scope? (such as a container element).

Reply:
changed it to observe the provided parent element argument if provided (i think i was using `document.body` temporarily to fix unexpected behaviour)